test/e2e/reboot.go

```diff
@@ -41,6 +41,9 @@ const (
 
 	// How long pods have to be "ready" after the reboot.
 	rebootPodReadyAgainTimeout = 5 * time.Minute
+
+	// How long pods have to be "ready" initially before a reboot.
+	rebootPodReadyInitialTimeout = 2 * time.Minute
```

Comment:
I'd be more inclined to increase podReadyBeforeTimeout (currently 20 sec) to, say 40 sec, for two reasons:

1. I can't see any reason why the pod startup time for these pods should be any different than other pods?

2. increasing the timeout 6x from 20 sec to 2 minutes seems excessive.